docs/good_first_issue.md

```
@@ -8,42 +8,42 @@ If you’re new to open source or this project, this guide will help you get sta
+
    - Comment on the issue:
      > “Hi, I’d like to work on this issue. Can I be assigned?”
    - A maintainer will assign it to you.
 3. **Fork & Clone**
-   
+
    ```bash
    git fork https://github.com/Rishabh-Bajpai/Chanakya-Local-Friend.git
    git clone https://github.com/YOUR_USERNAME/Chanakya-Local-Friend.git
    cd Chanakya-Local-Friend
    ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)